clip/convert.py

```diff
@@ -86,7 +49,13 @@ def should_keep_weight(key: str):
         default="mlx_model",
         help="Path to save the MLX model.",
     )
-
+    parser.add_argument(
+        "--dtype",
+        help="dtype for loading the torch model and input for quantization or saving the converted model. "
+        "The original weights are stored in bfloat16.",
```

Comment:
```suggestion
        help="The data type to save the converted model. "
```

I didn't get the ocmment about `bfloat16`?

Reply:
Sorry, it was copied from the mlx-lm converter.py and I forgot to update the description. I will clean up the description.